modules/ROOT/pages/salesforce/salesforce-connector-100-config-topics.adoc

```diff
@@ -400,11 +375,11 @@ When working with multiple currencies, be aware of which currency your sObject u
 
 === Limits on API Calls
 
-You need to know the rate limiting policy that applies to your account so that you do not exceed the number of allotted API calls per day.
+You must know the rate limiting policy that applies to your account so that you do not exceed the number of allotted API calls per day.
 
 === Opportunity Object
 
-When extracting data from an `Opportunity`, be aware that a "quarter" is not relative to a calendar year. A "quarter" in this context is relative to the financial year of the organization.
+When extracting data from an Opportunity, be aware that a "quarter" in this context is relative to the financial year of the organization instead of relative to the calendar year.
```

Comment:
```suggestion
When extracting data from an Opportunity object, be aware that a "quarter" in this context is relative to the financial year of the organization and not necessarily to the calendar year.
```

Reply:
This is a Salesforce object. I qualified it as such